ibis/spark/client.py

```diff
@@ -173,7 +162,7 @@ def _build_ast(self, expr, context):
         result = comp.build_ast(expr, context)
         return result
 
-    def _execute(self, stmt):
+    def _execute(self, stmt, results):
```

Comment:
The `results` parameter should be set to the base class's default value, which is `False`. If `results` is `False` then you don't return anything.